counterpoint/dist/counterpoint-vis.es.js

```diff
@@ -314,6 +314,15 @@ class b {
   changed() {
```

Comment:
Could you run `npm run build` after you're done making the other changes and make sure that you commit both `counterpoint-vis.es.js` and `counterpoint-vis.umd.js`? Both files should be changed.